Automattic/jetpack
# crm: Set up infrastructure to run tests in CI

## Proposed changes:

Import the tests from the old private repo, then set up the infrastructure to run them within our CI.

This doesn't make them actually pass, it seems there are a few things still broken, e.g.

* It expects to be in `wp-content/plugins/zero-bs-crm` but the CI puts it at `wp-content/plugins/crm`.
* `includes/lib/dompdf-2/vendor/` is missing.
* A few tests fail after fixing that too.

### Other information:


## Jetpack product discussion


p9dueE-6tU-p2
p1673631389693689-slack-C04J4UG3FLG

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Check the logs for the various PHP 7.2+ test runs, see if it runs tests for the plugin.
